hazelcast/src/test/java/com/hazelcast/instance/impl/DefaultAddressPickerTest.java

```diff
@@ -295,6 +295,15 @@ public void testPublicAddress_withSpecifiedHostAndPortViaProperty() throws Excep
         assertEquals(new Address(host, port), addressPicker.getPublicAddress(null));
     }
 
+
+    @Test(expected = IllegalArgumentException.class)
```

Comment:
It's so old-fashioned, to use the `expected` param in the `@Test` annotation. :stuck_out_tongue: 
Have you considered the more focused JUnit's `assertThrows` or AssertJ's `assertThatThrownBy`?

Reply:
updated in https://github.com/hazelcast/hazelcast/pull/24729/commits/b92e5a8c7a0150ce8504b5806fb47220bd6de359